plugins/Morpheus/stylesheets/general/_forms.less

```diff
@@ -92,10 +96,23 @@ select {
         // Smaller padding to match input's heights
         padding-top: 11px;
         padding-bottom: 10px;
+        border: 0px;
```

Comment:
Rather than customize locally, it would be better to change the global style of the alerts for consistency.